README.md

```diff
@@ -10,11 +10,17 @@ Easily imports your Gmail messages into Google Keep. Can also be used to transfe
 
 ## How to use
 
-1. First install the plugin from Chrome extension store: https://chrome.google.com/webstore/detail/lp-gmail-keep-importer/ingomolknmgnfbafknpkmklapabaednn
-2. Now visit the Google Keep website and log in, https://keep.google.com/#home
-3. Click the **LP G2K** icon on your Chrome extension bar to open the small popup of this extension 
-4. Type the label you want to import from Gmail.
-5. Click **Import GMail Label** and then if some email messages are ready to be imported click **Start importing**.
+Full instruction is here: https://developer.chrome.com/apps/tut_oauth
+
+*Long-story short:*
```

Comment:
I am sorry but this changes the indent of the extension. You can keep the changes in the documentation in your local fork if you want but I am not going to merge this.

Reply:
These instructions is to run the extension locally with your own keys, it's not how to use the extension when installed from the Chrome store.

If you want to add these, please rebase from latest, and add the instructions in a separate `How to run locally with a fork` section.

Reply:
But your extension isn't available in my Chrome store :(

Reply:
https://chrome.google.com/webstore/detail/lp-gmail-keep-importer/ingomolknmgnfbafknpkmklapabaednn

I re-published it yesterday. Google had it removed after some changes in their policies.